__DEFINES/__compile_options.dm

```diff
@@ -36,18 +36,18 @@
 
 // Toggles several features, explained in their respective comments.
 // You can turn those on and off manually if you prefer, instead of setting this
-#define DEVELOPER_MODE 0
+#define DEVELOPER_MODE 1
```

Comment:
delete this

Reply:
goddamnit, i was wondering where my compile options went